src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -673,7 +748,15 @@ const StudentApplication = ({ userProfile, authentication }) => {
                     <Card>
                       <CardHeader title="Off-Campus Work Study" className="card-header" />
                       <CardContent>
-                        <Typography variant="body1">Placeholder text</Typography>
+                        <offCampusSection
+                          offCampusApplicantList={offCampusApplicants}
+                          availableApplicants={applicants}
```

Comment:
You are using this `offCampusApplicantList` to do exactly what the `applicants` state variable was designed to do. There is already support for this data structure built into the frontend and the backend, so why create another array variable to hold that same information? 
We should be thinking of the applicants as being almost like a key-value pair, where the StudentProfileInfo object is the unique key and the name of the off campus program department is one of the values

Reply:
If you make sure to merge in the latest changes from `apartapp-develop`, I think you should be able to check the `@typedef` in `services/housing.js` for the properties that have already been assigned to the objects in the `applicants` array

Reply:
Ohhhhh. That makes sense. So it should be less of having a separate list of off-campus applicants and more of checking that applicants list for which ones program values are being updated.